src/python-common/ceph/deployment/drive_group.py

```diff
@@ -177,6 +177,7 @@ def __init__(self,
                  extra_container_args=None,  # type: Optional[List[str]]
                  data_allocate_fraction=None,  # type: Optional[float]
                  method=None,  # type: Optional[OSDMethod]
+                 crush_device_class=None, # type: Optional[str]
```

Comment:
```suggestion
                 crush_device_class=None,  # type: Optional[str]
```
for flake8 (which is why make check is failing)
```
ceph/deployment/drive_group.py:180:42: E261 at least two spaces before inline comment
```